api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -1249,11 +1249,10 @@ private void enablePatientRegistration() {
 
         // Show additional identifiers (from form section "patient-identification-section")
         //   - in Mexico
-        //   - in non-mental-health Haiti if the additionalHaitiIdentifiers feature toggle is enabled
+        //   - in Haiti if configured for HIV
         if (config.getCountry().equals(ConfigDescriptor.Country.MEXICO) || (
-                featureToggles.isFeatureEnabled("additionalHaitiIdentifiers") &&
                         config.getCountry().equals(ConfigDescriptor.Country.HAITI) &&
-                        !ConfigDescriptor.Specialty.MENTAL_HEALTH.equals(config.getSpecialty()))) {  // reversed to make this null safe
+                        ConfigDescriptor.Specialty.HIV.equals(config.getSpecialty()))) {  // reversed to make this null safe
```

Comment:
```suggestion
                        ConfigDescriptor.Specialty.HIV.equals(config.getSpecialty()))) {
```

Reply:
Why are you suggesting I remove that comment?  Is that what you are suggesting?

Reply:
Yes, because the comment no longer is true, right? It is no longer reversed (the `!` is no longer present in the condition).

Reply:
I don't think it's referring to the !.  I think it's referring to putting the constant before the variable in the equals comparison, so that you are never calling equals on null.

Reply:
Oh yeah, yep, definitely. Thanks.